docs/dev/userland.md

```diff
@@ -281,17 +281,18 @@ gmake sample-manifest
 
 to generate a manifest from the list of installed files.
 
-Copy the file `build/manifest-generated.p5m` to `$(COMPONENT_NAME).p5m` and edit it:
+Copy the file `manifests/sample-manifest.p5m` to `$(COMPONENT_NAME).p5m` and edit it:
 
 * Add your name as a contributor
 * Remove unused entries from the manifest:
     * directories: `:%g/^dir/d` (Vim)
     * static libs: `:%g/.a$/d` (Vim)
-    * libtool files:: `:%g/.la$/d` (Vim)
+    * libtool files: `:%g/.la$/d` (Vim)
     * Python *.pyc: `:%g/.pyc$/d` (Vim)
 
 For some components, specific rules need to be applied: they can be implemented with *transforms*.
-Some example can be found in the directory with the same name at the root directory of oi-userland.
+Some transforms, located in the directory `transforms` at the root of oi-userland, are applied automatically by various Makefile targets defined in `make-rules/ips.mk`.
```

Comment:
```suggestion
Transforms, located in the directory `transforms` and in various include files in the `make-rules` directory at the root of oi-userland, are applied automatically by various Makefile targets.
```

Reply:
Transforms are not located in "various include files in the `make-rules` directory", needs better wording.